.github/CODEOWNERS

```diff
@@ -1,5 +1,5 @@
 * @bcumming @msimberg @RMeli
 docs/services/firecrest @jpdorsch @ekouts
-docs/software/communication @msimberg
+docs/software/communication @biddisco @Madeeks @msimberg
```

Comment:
@Madeeks @biddisco I've tentatively added you as codeowners. Is this ok for you? This means you'd get requested as reviewers whenever changes under `docs/software/communication` is touched.

Perhaps @teojgo, @boeschf are also candidates? If interested, please comment.

Reply:
It's ok for me. I don't know much about the inner workings of the comm libs, but I can share my experiences from "the outside".

Reply:
Thanks @Madeeks! I think you know plenty enough, and you don't have to know _everything_...

Reply:
I removed @biddisco for now.